VERSION

```diff
@@ -1 +1 @@
-1.32.0
\ No newline at end of file
+1.34.0
```

Comment:
did we miss a version?

Reply:
might have accidentally bumped twice, I can fix